ci/jenkinsfile.groovy

```diff
@@ -98,13 +111,38 @@ stage('Build') {
       node('CPU-SMALL') {
         ws(per_exec_ws('mlc-llm-build')) {
           init_git(true)
-          sh(script: "ls", label: 'debug')
           sh(script: 'git clone --recursive https://github.com/mlc-ai/package', label: 'Clone mlc-ai/package')
           sh(script: "${pkg_cuda} ./ci/task/build_lib.sh", label: 'Build MLC LLM runtime')
           sh(script: "${pkg_cuda} ./ci/task/build_wheel.sh", label: 'Build MLC LLM wheel')
+          pack_lib('mlc_llm_runtime_cuda', 'wheels/*.whl')
           sh(script: "${pkg_cuda} ./ci/task/build_clean.sh", label: 'Cleanups after build')
         }
       }
     }
   )
 }
+
+def compile(target) {
+  node('CPU-SMALL') {
+    ws(per_exec_ws('mlc-llm-compile-models')) {
+      init_git(false)
+      unpack_lib('mlc_llm_runtime_cuda', 'wheels/*.whl')
+      withCredentials([string(
+        credentialsId: 'hf_download_token',
+        variable: 'HF_TOKEN',
+      )]) {
```

Comment:
We don't actually need to download anything from huggingface in compilation testing, in fact, the lines below should work:

```bash
mlc_chat compile llama2_70b --quantization q4f16_1 --device cuda:0 --output /tmp/1.so
```

where `llama2_70b` corresponds to the names here: https://github.com/mlc-ai/mlc-llm/blob/main/python/mlc_chat/model/model_preset.py